inst/odin/carehomes.R

```diff
@@ -1460,7 +1527,9 @@ update(react_pos) <- sum(new_T_PCR_pos[2:18, , , ])
 
 ## prob_strain gives probability of an infection in group i being of strain j
 initial(prob_strain[, ]) <- 0
-dim(prob_strain) <- c(n_groups, n_strains)
+dim(prob_strain) <- c(n_groups, n_real_strains)
```

Comment:
I think given what we have decided to do for the Rt calculation we could drop the n_group dimension. I've tried to do this in a later commit but didn't go very far